bw2analyzer/lci.py

```diff
@@ -1,14 +1,60 @@
 import pandas as pd
-from bw2data import get_activity
+from bw2data import Database
 from bw2calc import LCA
+from typing import List, Union
 
 
-def get_labeled_inventory(lca: LCA) -> pd.DataFrame:
+def _get_dependent_database_names(lca: LCA) -> List[str]:
+    databases = []
+    for key in lca.remapping_dicts.keys():
+        databases += [key[0] for key in lca.remapping_dicts[key].values()]
+    return pd.unique(databases)
+
+
+def _load_database_metadata(
+    lca: LCA, cols: Union[None, List[str], str] = None
```

Comment:
Best practice to use `Optional` instead of `Union[None` (I just learned this myself).